src/fingers/action_runner.cr

```diff
@@ -124,7 +124,7 @@ module Fingers
 
     # This takes care of some path expansion weirdness when opening paths that start with ~ in MacOS
     def expanded_match
-      return match unless action == ":open:"
+      return match unless action == ":open:" && match.starts_with?("~")
```

Comment:
I find `unless` statements with multiple conditions hard to read.

A private method can help

```crystal
def expanded_match
  return match unless should_expand_match?
      
  # ...
end

def should_expand_match?
  action == ":open:" && match.starts_with?("~")
end
```

Reply:
Done!